src/pages/learn/index.mdx

```diff
@@ -52,8 +56,52 @@ Could produce the following JSON result:
 
 ```json
 {
-  "me": {
-    "name": "Luke Skywalker"
+  "data": {
+    "me": {
+      "name": "Luke Skywalker"
+    }
   }
 }
 ```
+
+With even a simple query, we can see some of the key features that make GraphQL so powerful. The client can make queries to the API that mirror the structure of the data that they need and then receive just that data in the expected shape with a single request—and without having to worry about what underlying data sources provided it.
+
+## Evolve your API without versioning
+
+Client requirements change over time and GraphQL allows your API to evolve in response to those needs and without the overhead of managing different API versions. For example, if a new feature calls for more specific name values to be available, then the `User` type could be updated as follows:
+
+```graphql
+type User {
+  fullName: String
+  nickname: String
+  name: String @deprecated(reason: "Use `fullName`.")
+}
+```
+
+GraphQL will continue to provide data as expected for the deprecated `name` field until it's determined that clients have updated their existing queries and the older field can be safely removed.
+
+## Try it out!
+
+The best way to learn GraphQL is to start writing queries. The query editors used throughout this guide are **interactive**, so try adding an `id` and `appearsIn` fields to the `hero` object to see an updated result:
+
+```graphql
+# { "graphiql": true }
+{
+  hero {
+    name
+    # add additional fields here!
+  }
+}
+```
+
+<Callout type="info">
+The examples in this guide are based on [a modified version of the SWAPI GraphQL schema](https://github.com/graphql/graphql.github.io/blob/source/src/components/marked/swapi-schema.tsx). Because these queries are designed for illustrative purposes, they will not run on the full version of the SWAPI GraphQL API due to differences between the two schemas. [You can try the full version of the API here.](https://graphql.org/swapi-graphql/)
+</Callout>
+
+## Next steps
+
+Now that you know some key GraphQL concepts, you're ready to learn about all of the different aspects of its [type system](/schema/).
+
+If you're already familiar with GraphQL and would like to read documentation on how to build a GraphQL service, then there are libraries to help you implement GraphQL in [many different languages](/community/tools-and-libraries/?tags=server). There are also many libraries available that allow [client applications to query existing GraphQL APIs](/community/tools-and-libraries/?tags=client).
```

Comment:
If someone is already familiar with GraphQL it seems unlikely they would read this far - perhaps this should be in a callout at the top of the page? I'm not super familiar with Nextra, but in Docusaurus I'd use an "info" [admonition](https://docusaurus.io/docs/markdown-features/admonitions).

Reply:
Ok, I'll move this back to the top.